src/Model/List/ListDecorator.php

```diff
@@ -5,39 +5,36 @@
 use SilverStripe\Model\ModelData;
 use LogicException;
 use Traversable;
+use BadMethodCallException;
 
 /**
  * A base class for decorators that wrap around a list to provide additional
  * functionality. It passes through list methods to the underlying list
  * implementation.
  *
- * @template TList of SS_List&Sortable&Filterable&Limitable
```

Comment:
Don't remove this! This represents the underlying list. It was correct, just need to remove the `&Sortable`etc from the end.

Reply:
Updated